homeassistant/components/myuplink/diagnostics.py

```diff
@@ -0,0 +1,46 @@
+"""Diagnostics support for myUplink."""
+from __future__ import annotations
+
+from typing import Any
+
+from myuplink import MyUplinkAPI
+
+from homeassistant.components.diagnostics import async_redact_data
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+
+TO_REDACT = {"access_token", "refresh_token", "serialNumber"}
```

Comment:
I don't see the serial number in the mock config entry data. Where is the serial number redacted?

Reply:
Good catch!
The serialNumber(s) are unredacted.
The redact has disappeared somehow. I must have mixed up version of diagnostics.py.
I'll be back.

Reply:
Fixed.